Podfile

```diff
@@ -114,4 +126,69 @@ post_install do |installer_representation|
     justLikeSed(filename , '-framework "CoreTelephony" ' , '')
   end
 
+  # ShortcutRecorder sets a weird PRODUCT_BUNDLE_IDENTIFIER
+  # change to ours
+  SRCommonFileName='Pods/ShortcutRecorder/Sources/ShortcutRecorder/SRCommon.m'
+
+  puts "chmod +w: Pods/ShortcutRecorder/Sources/ShortcutRecorder/SRCommon.m"
+
+  if system("chmod +w Pods/ShortcutRecorder/Sources/ShortcutRecorder/SRCommon.m")
+    puts "chmod success"
+  else
+    puts "chmod failed"
+  end
+
+  puts "Replacing 'com.kulakov.ShortcutRecorder' with 'com.sequel-ace.sequel-ace'"
+
+  justLikeSed(SRCommonFileName, 'com.kulakov.ShortcutRecorder', 'com.sequel-ace.sequel-ace')
+
+  # ShortcutRecorder has loads of resources that we don't need
+  # remove them
+  puts "Removing uneeded resources"
+
+  resourcesShellFileName='Pods/Target Support Files/Pods-Sequel Ace/Pods-Sequel Ace-resources.sh'
+
+  justLikeSed(resourcesShellFileName, 'install_resource "${PODS_ROOT}/ShortcutRecorder/ATTRIBUTION.md"', '')
+  justLikeSed(resourcesShellFileName, 'install_resource "${PODS_ROOT}/ShortcutRecorder/LICENSE.txt"', '')
+
+  arr = ['ca', 'cs', 'de', 'el', 'es', 'fr', 'it', 'ja', 'ko', 'nb', 'nl', 'pl', 'pt-BR', 'pt', 'ro', 'ru', 'sk', 'sv', 'th', 'zh-Hans', 'zh-Hant']
+
+  arr.each do |lang|
+    search = 'install_resource "${PODS_ROOT}/ShortcutRecorder/Sources/ShortcutRecorder/Resources/' + lang + '.lproj"'
+    justLikeSed(resourcesShellFileName, search, '')
+  end
+
+  # ShortcutRecorder sets default font to 13
+  # we want 11, so change it here
+  # There must be a way to do this in code, but
+  # I haven't figured it out yet
+  srMojaveInfoFile='Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-mojave-info.dataset/info.json'
+
+  puts "chmod +w: Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-mojave-info.dataset/info.json"
+
+  if system("chmod +w Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-mojave-info.dataset/info.json")
+    puts "chmod success"
+  else
+    puts "chmod failed"
+  end
+
+  puts "Replacing '13.0' with '11.0'"
+
+  justLikeSed(srMojaveInfoFile, '13.0', '11.0')
+
+  srYosemiteInfoFile='Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-yosemite-info.dataset/info.json'
+
+  puts "chmod +w: Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-yosemite-info.dataset/info.json"
+
+  if system("chmod +w Pods/ShortcutRecorder/Sources/ShortcutRecorder/Resources/Images.xcassets/sr-yosemite-info.dataset/info.json")
+    puts "chmod success"
+  else
+    puts "chmod failed"
```

Comment:
I do agree we may want to try to find a way to do this in code?

Reply:
Still trying....

Reply:
There's no interface to change the attributes. Need to subclass. Or edit the info.json.....